junit-platform-launcher/src/main/java/org/junit/platform/launcher/listeners/LoggingListener.java

```diff
@@ -73,7 +74,13 @@ public static LoggingListener forJavaUtilLogging(Level logLevel) {
 	 * @see #forJavaUtilLogging()
 	 * @see #forJavaUtilLogging(Level)
 	 */
-	public LoggingListener(BiConsumer<Throwable, Supplier<String>> logger) {
+	public static LoggingListener forBiConsumer(BiConsumer<Throwable, Supplier<String>> logger) {
```

Comment:
@sbrannen I realise  that it's not clear to me from this javadoc what the BiConsumer parameter consumes, so I wonder if having a `@param` description for the BiConsumer, with additional notes on the two effective parameters of the BiConsumer itself, would improve clarity here.

Reply:
Good point! Addressed in 838cb09.